frontend/package.json

```diff
@@ -56,94 +59,59 @@
     "focus-visible": "^5.2.0",
-    "glob": "^8.0.1",
-    "node-html-parser": "^6.0.0",
-    "nuxt": "^2.17.3",
     "pinia": "^2.1.7",
-    "portal-vue": "^2.1.7",
-    "postcss-focus-visible": "^9.0.0",
-    "prom-client": "^14.0.1",
-    "rfdc": "^1.3.0",
+    "postcss-focus-visible": "^9.0.1",
+    "rfdc": "^1.4.1",
     "seeded-rand": "^2.0.1",
-    "throttle-debounce": "^5.0.0",
-    "uuid": "^9.0.1",
-    "vue": "^2.7.16",
-    "vue-i18n": "^8.26.7",
-    "vue-plausible": "^1.3.2"
+    "throttle-debounce": "^5.0.2",
+    "uuid": "^10.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.24.4",
-    "@babel/parser": "^7.24.4",
-    "@babel/preset-env": "^7.24.4",
-    "@babel/preset-typescript": "^7.24.1",
-    "@itsjonq/remake": "^2.0.0",
-    "@nuxt/types": "^2.17.3",
-    "@nuxt/typescript-build": "^3.0.2",
-    "@nuxtjs/storybook": "^4.3.2",
-    "@pinia/testing": "^0.1.0",
-    "@playwright/test": "1.45.0",
-    "@tailwindcss/typography": "^0.5.10",
-    "@testing-library/dom": "^9.3.4",
-    "@testing-library/jest-dom": "^6.4.2",
+    "@babel/parser": "^7.24.8",
+    "@nuxt/test-utils": "^3.13.1",
+    "@nuxtjs/i18n": "^8.3.1",
+    "@playwright/test": "1.45.2",
     "@testing-library/user-event": "^14.5.2",
-    "@testing-library/vue": "^5.9.0",
-    "@types/express-useragent": "^1.0.2",
-    "@types/jest": "^29.5.12",
-    "@types/node": "^20.12.11",
-    "@types/throttle-debounce": "^5.0.0",
-    "@types/uuid": "^9.0.6",
-    "@vue/test-utils": "^1.1.3",
-    "@vue/vue2-jest": "^29.2.6",
-    "adm-zip": "^0.5.10",
-    "autoprefixer": "^10.4.16",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-jest": "^29.7.0",
-    "babel-loader": "8.3.0",
-    "caniuse-lite": "^1.0.30001620",
-    "chokidar": "^3.5.3",
-    "comment-json": "^4.2.3",
-    "css-loader": "^5.2.7",
-    "jest": "^29.7.0",
-    "jest-environment-jsdom": "^29.7.0",
-    "jest-fail-on-console": "^3.2.0",
-    "jest-transform-stub": "^2.0.0",
-    "npm-run-all2": "^6.1.2",
-    "postcss": "^8.4.31",
-    "qs": "^6.11.0",
-    "rimraf": "^5.0.0",
-    "tailwindcss": "^3.3.5",
-    "talkback": "^4.0.0",
-    "ts-node": "^10.9.1",
-    "typescript": "^5.5.2",
-    "vue-demi": "^0.14.7",
-    "vue-i18n-extract": "^2.0.7",
-    "vue-loader": "^15.11.1",
-    "vue-server-renderer": "^2.7.16",
-    "vue-template-compiler": "^2.7.16",
-    "vue-tsc": "2.0.24",
-    "webpack": "^4.46.0"
+    "@testing-library/vue": "^8.1.0",
+    "@types/node": "^20.14.11",
+    "@vitest/coverage-v8": "^2.0.4",
+    "@vitest/ui": "^2.0.4",
+    "@vue/test-utils": "^2.4.6",
+    "adm-zip": "^0.5.14",
+    "chokidar": "^3.6.0",
+    "core-js": "^3.37.1",
+    "eslint-plugin-jsonc": "^2.16.0",
+    "jsdom": "^24.1.0",
+    "node-html-parser": "^6.1.13",
+    "npm-run-all2": "^6.2.2",
+    "nuxt": "3.12.4",
+    "rimraf": "^6.0.1",
+    "talkback": "^4.2.0",
+    "typescript": "5.5.3",
+    "vitest": "^2.0.4",
+    "vitest-dom": "^0.1.1",
+    "vue": "3.4.33",
+    "vue-router": "^4.4.0",
```

Comment:
Just noting: This release includes a partial fix for the memory leak issue! https://github.com/vuejs/router/pull/2137